velox/functions/sparksql/tests/DecimalCompareTest.cpp

```diff
@@ -128,7 +128,7 @@ TEST_F(DecimalVectorFunctionsTest, gt) {
       makeNullableFlatVector<bool>({false, std::nullopt, false, false}));
 }
 
-TEST_F(DecimalVectorFunctionsTest, gte) {
```

Comment:
Nice refactor. Would you extract it into a separate PR?

Reply:
Sure. Opened https://github.com/facebookincubator/velox/pull/7425.